web/modules/custom/mymodule/src/Form/SimpleForm.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+namespace Drupal\mymodule\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Locale\CountryManager as CountryManager;
+use Drupal\Core\Messenger\MessengerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Simpleform class using simple form api.
+ */
+class SimpleForm extends FormBase {
+
+  /**
+   * Messenger variable.
+   *
+   * @var object
```

Comment:
@anitha-innoraft we should always specify the type of property/object. For e.g. Here `$messanger` is a type of `Drupal\Core\Messenger\MessengerInterface`. We should specify the type instead of `object`

Reply:
Ok. Fixed it